tests/compile.sh

```diff
@@ -60,6 +60,29 @@ cd "$PATHTR/../NEMS"
 COMPONENTS="FMS,FV3"
 if [[ "${MAKE_OPT}" == *"CCPP=Y"* ]]; then
   COMPONENTS="CCPP,$COMPONENTS"
+
+  # Check if suites argument is provided or not
+  set +ex
+  TEST=$( echo $MAKE_OPT | grep -e "SUITES=" )
+  if [[ $? -eq 1 ]]; then
+    echo "No suites argument provided, compiling all available suites ..."
+    # Loop through all available suite definition files and extract suite names
+    SDFS=(../FV3/ccpp/suites/*.xml)
+    SUITES=""
+    for sdf in ${SDFS[@]}; do
+      suite=${sdf#"../FV3/ccpp/suites/suite_"}
+      suite=${suite%".xml"}
+      SUITES="${SUITES},${suite}"
+    done
+    # Remove leading comma
+    SUITES=${SUITES#","}
+  else
+    SUITES=$( echo $MAKE_OPT | sed 's/.* SUITES=//' | sed 's/ .*//' )
+  fi
+  echo "Compiling suites ${SUITES}"
+  MAKE_OPT="${MAKE_OPT} SUITES=${SUITES}"
```

Comment:
I realize now that this will duplicate the `SUITES=...` argument in case it is already provided in the `MAKEOPT` string. This doesn't hurt, though, because the NEMS build system only extracts it once before calling `ccpp_prebuild.py`.